frontend/src/services/product.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ *
+ * @param {String} userId - the unique identifier for the user
+ * @param {Object} set - an object containing any number of properties to bind with this user, these can be overridden
+ * @param {Object} setonce - an object containing any number of properties to bind with this user, these can never be overridden
+ */
+function identify (userId, set, setonce) {
+    window.posthog?.identify(userId, set, setonce)
+}
+
+/**
+ *
+ * @param {String} event - the name/identifier of the event
+ * @param {Object} properties - an object containing any number of properties to bind with this event, this can also contain
+ * a proeprty $set or $set_once which in turn can be an object to bind properties to the associated user
+ * @param {Object} groups - ties a given 'group' to the event. Optional keys: 'team', 'application', 'instance', 'device'
+ */
+function capture (event, properties, groups) {
+    if (!properties) {
+        properties = {}
+    }
+    if (groups) {
+        properties.$groups = groups
+    }
+    window.posthog?.capture(event, properties)
+}
+
+/**
+ *
+ * @param {String} type - team | application | instance | device
+ * @param {String} id - the associated id to this group type
+ * @param {Object} proeprties - a JSON object defining features for this group, all can be overriden
+ */
+function groupUpdate (type, id, properties) {
+    capture('$groupidentify', {
```

Comment:
for context on this: https://github.com/PostHog/posthog-js/issues/600

currently, in the client-side JS library, we cannot call _just_ a `groupIdentify`, the client JS `group()` function currently sets a session variable such that _all_ future events are bound to that group, which we do not want. In speaking with the dev team at PostHog (see the linked Slack thread) they informed me that I can just do this instead.